test/integration/style-spec/tests/bad-color.output.json

```diff
@@ -10,5 +10,17 @@
   {
```

Comment:
I noticed that there were both `*.output-api-supported.json` and `*.output.json` files in the tests, but was unable to find references to `output-api-supported` or `api-supported` in the codebase, other than a changelog entry saying `--mapbox-api-supported` had been removed from `gl-style-validation` in 14.0.0. Are the `.output-api-supported.json` files still used, or subject for removal in a later PR?

Reply:
I can't find anything related to this in the history of the repo, I'm not sure they were ever used, and also from a very shallow test I made these looks duplicates of the output files, so I would recommend removing them.

Reply:
Sure. Remove in this PR or shall I make a separate PR?

Reply:
A separate PR seems like the right approach.